docs/workflows.rst

```diff
@@ -0,0 +1,153 @@
+================
+GitHub Workflows
+================
+
+All Python documentation translation repositories are hosted on `GitHub <https://github.com>`_.
+GitHub provides a very useful `CI <https://en.wikipedia.org/wiki/Continuous_integration>`_
+system, *GitHub Actions*, which runs *workflows*. Workflows are particularly useful
+for testing projects, in translation repos they can be used to update translations
+from Transifex, lint translated files, and test build the translated documentation.
+
+This repository provides three sample workflows, a Linting workflow, a Test Build
+workflow, and a workflow for pulling translations from Transifex. They can be found
+in the `sample-workflows <https://github.com/python-docs-translations/transifex-automations/tree/main/sample-workflows>`_
+and explanations of how they work and what you need to do to set them up can be found below.
+
+.. seealso::
+   - `GitHub docs: Writing workflows <https://docs.github.com/en/actions/writing-workflows>`_
+   - :doc:`Setting up Transifex <commands.rst>`
+
+Running Workflows
+-----------------
+
+To run on of our provided workflows (or any workflow), put the file in the
+``.github/workflows`` folder and it will run automatically depending on how it
+is configured.
+
+General Overview
+----------------
+
+.. code-block:: workflow
+
+   name: Linting Workflow
+
+The name can be freely configured and will be displayed in the "Actions" tab.
+
+.. code-block:: workflow
+
+   on:
+      schedule:
+         - cron: '0 0 * * *'
+
+Using `cron <https://en.wikipedia.org/wiki/Cron>`_, the frequency of the workflow
+runs can be set. In the sample workflows it is configured to ``'0 0 * * *'``,
+meaning it will run once daily.
+
+.. code-block:: workflow
+
+   on:
+      ...
+      push:
+         branches:
+            - '*'
+      workflow_dispatch:
+
+The workflow can also be set to run under different conditions, in the case of the
+sample workflows it is set to run on a ``push`` to any branch, and can be run
+manually due to the ``workflow_dispatch`` option.
+
+.. code-block:: workflow
+
+   jobs:
+
+All jobs or sections under ``jobs:`` will be run. The sample workflows can be
+merged into one by combining all of their jobs. (Note: there must only be one `jobs:`)
+
+
+Transifex Pull Workflow
+-----------------------
+
+How to configure the `Transifex Pull Workflow <https://github.com/python-docs-translations/transifex-automations/blob/main/sample-workflows/transifex-pull.yml>`_.
+
+This workflow automatically pulls updated translations from Transifex, commits
+them to the repository, and pushes them back to the relevant branch, if
+significant changes are detected.
+
+Ensure the ``TX_TOKEN`` secret is configured in your repository with your Transifex API token.
+
+In the workflow, replace all instances of ``XX`` with your ITFL language code.
```

Comment:
How about add examples in here? At least 2, one without country part and other with.